### PR TITLE
Style tweaks: notification on toggling style tweak with a gesture

### PR DESCRIPTION
**Pull request by hius07**
Closes https://github.com/koreader/koreader/issues/10667.

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10674)

### CODE REVIEW

File: frontend/apps/reader/modules/readerstyletweak.lua [hunk 1]
```diff
@@ -726,6 +727,7 @@ function ReaderStyleTweak:onToggleStyleTweak(tweak_id, item)
         else
             self.doc_tweaks[tweak_id] = nil
         end
+        text = _("Disabled style tweak: ")
```
Comment (poire-z) [inline frontend/apps/reader/modules/readerstyletweak.lua:730]: @Frenzie will confirm if there is a risk having translatable strings ending with a space - and translator missing it in their translation.

(These notifications will probably be long and end up truncated.)

Comment (Frenzie) [reply frontend/apps/reader/modules/readerstyletweak.lua:730]: Yes, at a glance it'd be better to use the template function right here and in the else with %1 and `self.tweaks_in_dispatcher[tweak_id]` as an argument.

File: frontend/apps/reader/modules/readerstyletweak.lua [hunk 2]
```diff
@@ -727,7 +727,7 @@ function ReaderStyleTweak:onToggleStyleTweak(tweak_id, item, no_notification)
         else
             self.doc_tweaks[tweak_id] = nil
         end
-        text = _("Disabled style tweak: ")
+        text = T(_("Off: %1"), self.tweaks_in_dispatcher[tweak_id])
```
Comment (poire-z) [inline frontend/apps/reader/modules/readerstyletweak.lua:730]: I guess that for such short and standard On/Off:, we should add some translation context and/or a @ translator comment.

Comment (mergen3107) [reply frontend/apps/reader/modules/readerstyletweak.lua]: Can we just make these notifications always in two lines? First line is “Style tweak enabled/disabled:” in any language, second line is the css name.

Comment (hius07) [reply frontend/apps/reader/modules/readerstyletweak.lua]: Currently notification is a TextWidget
https://github.com/koreader/koreader/blob/b1c90260e40dc5b1f6c58dc9cac9b26786572015/frontend/ui/widget/textwidget.lua#L2

Comment (poire-z) [reply frontend/apps/reader/modules/readerstyletweak.lua]: Yes, I have thought about having it multilines in the past, but gave up. We don't really need it to be more than one line, we're mostly fine elsewhere - and most people don't care about notifications, so the smaller the better :)

Comment (Frenzie) [reply frontend/apps/reader/modules/readerstyletweak.lua]: I do think it's a good suggestion all around. Concatenating `.. "\n" ..` (or maybe two) bothers translators the minimum amount and the display would likely be better too. Except for that single line notification aspect. :-)

Comment (Frenzie) [reply frontend/apps/reader/modules/readerstyletweak.lua]: But yes, also notifications should be as small as possible, forgot about that for a second.

**Review by poire-z — APPROVED** [review]
We'll see how many people have a profile enabling 14 tweaks and get a notification stack overflow :)

**Comment by mergen3107** [comment]
Stack Overflow on GitHub? That’s something new :D

**Pull request merged**